packages/host/src/components/Initialise/Initialise.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect } from 'react';
+import {
+  useTranslation,
+  LoadingButton,
+  Box,
+  Typography,
+  AlertIcon,
+  useHostContext,
+  SaveIcon,
+} from '@openmsupply-client/common';
+import { LoginTextInput } from '../Login/LoginTextInput';
+import { InitialiseLayout } from './InitialiseLayout';
+import { useInitialiseForm } from '../hooks';
+
+export const Initialise = () => {
+  const t = useTranslation('app');
+  const { setPageTitle } = useHostContext();
+
+  const {
+    isValid,
+    isLoading,
+    password,
+    siteId,
+    url,
+    username,
+    onSave,
+    setPassword,
+    setUsername,
+    setSiteId,
+    setUrl,
+    error,
+  } = useInitialiseForm();
+
+  useEffect(() => {
+    setPageTitle(`${t('app.initialise')} | ${t('app')} `);
+  }, []);
+
+  return (
+    <InitialiseLayout
+      UsernameInput={
+        <LoginTextInput
+          fullWidth
+          label={t('label.settings-username')}
+          value={username}
+          disabled={isLoading}
+          onChange={e => setUsername(e.target.value)}
+          inputProps={{
+            autoComplete: 'username',
+          }}
+          autoFocus
+        />
+      }
+      PasswordInput={
+        <LoginTextInput
+          fullWidth
+          label={t('label.settings-password')}
+          type="password"
+          value={password}
+          disabled={isLoading}
+          onChange={e => setPassword(e.target.value)}
+          inputProps={{
+            autoComplete: 'current-password',
+          }}
+        />
+      }
+      UrlInput={
+        <LoginTextInput
+          fullWidth
+          label={t('label.settings-url')}
+          value={url}
+          disabled={isLoading}
+          onChange={e => setUrl(e.target.value)}
+        />
+      }
+      SiteIdInput={
```

Comment:
hoping that we could remove this at some point - so haven't gone very fancy with the numeric input handling.